frontend/src/metabase/search/components/SearchSidebar/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -4,7 +4,7 @@ import { useLayoutEffect, useRef, useState } from "react";
 import { t } from "ttag";
```

Comment:
This file and the next are just changing some type names and things so I can have a toggle filter (which we'll also need for the SQL searching)